components/Typography/src/MDCTypography.m

```diff
@@ -136,6 +136,20 @@ + (CGFloat)buttonFontOpacity {
   return MDCTypographyStandardOpacity;
 }
 
++ (BOOL)isLargeForContrastRatios:(nullable UIFont *)font {
```

Comment:
nonnull UIFont to match the font loaders?

Otherwise, shortcircuit return NO if font == nil.